src/inputs/dropdownMenu/dropdownMenu.jsx

```diff
@@ -245,7 +247,7 @@ function DropdownMenu(props) {
         onDropdownMenuReposition();
 
         return () => {
-            if (!isOpen) {
+            if (isOpen) {
```

Comment:
Can we double-check this?
See https://github.com/saddlebackdev/react-cm-ui/commit/91ba55053be35f638804efca7a97f57c904298fa#diff-cd14cd4dc11800d818e7e59512fcad5a4ec75c5b23cfdaa368b9db636b6165c0L233-R248 / https://github.com/saddlebackdev/react-cm-ui/pull/438.

Reply:
@groberts314 I've double checked, with current `(!isOpen)`, listeners are not removed on dev, stage, stage2. If needed, I can decompose the explanation of why it's not working from the description into an ordered list